src/internal_events/docker.rs

```diff
@@ -0,0 +1,194 @@
+use super::InternalEvent;
+use bollard::errors::Error;
+use chrono::ParseError;
+use metrics::counter;
+
+#[derive(Debug)]
+pub struct DockerEventReceived<'a> {
+    pub byte_size: usize,
+    pub container_id: &'a str,
+}
+
+impl<'a> InternalEvent for DockerEventReceived<'a> {
+    fn emit_logs(&self) {
+        trace!(
+            message = "received one event.",
+            byte_size = %self.byte_size,
+            container_id = %self.container_id
+        );
+    }
+
+    fn emit_metrics(&self) {
+        counter!("events_processed", 1,
+                 "component_kind" => "source",
+                 "component_name" => "docker",
+        );
+        counter!("bytes_processed", self.byte_size as u64,
+                 "component_kind" => "source",
+                 "component_name" => "docker",
+        );
+    }
+}
+
+#[derive(Debug)]
+pub struct DockerContainerEventReceived<'a> {
+    pub container_id: &'a str,
+    pub action: &'a str,
+}
+
+impl<'a> InternalEvent for DockerContainerEventReceived<'a> {
+    fn emit_logs(&self) {
+        debug!(
```

Comment:
```suggestion
        trace!(
```

Previously level for this level was `trace`.

Reply:
It was, but it's more important than `DockerEventReceived` so it would be nice to have it more visible than those, but it is less important than `DockerContainer*watch`, so `debug` seems like a nice middle.